src/js/tasks-function.js

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly remove the linters disable line. You can put your classes in separate files and then import them where necessary.